vector-config/src/main/res/values/config-settings.xml

```diff
@@ -45,7 +45,7 @@
     <bool name="settings_labs_deferred_dm_visible">true</bool>
     <bool name="settings_labs_deferred_dm_default">true</bool>
     <bool name="settings_labs_thread_messages_default">false</bool>
-    <bool name="settings_labs_new_app_layout_default">false</bool> <!-- Tchap: keep the previous layout by default -->
+    <bool name="settings_labs_new_app_layout_default">true</bool>
```

Comment:
I'm not sure we want to activate the new layout on this PR, please keep it to false for the moment